onnxruntime/contrib_ops/cpu/qlinear_lookup_table.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+void QLinearLookupTableTransform(const uint8_t* x, const uint8_t* table, uint8_t* y, size_t n);
```

Comment:
>QLinearLookupTableTransform [](start = 5, length = 27)

there is not need to expose this function

Reply:
Yeah, could hide it currently.